### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by ShinhyoBelok**
# Changes 🛠️ 

* Add Home page
* Add Home page CSS
* Add houses display 3 by 3
* Add houses fetch function
* Add dynamic arrow functionalities

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from './App';
+import store from './redux/configureStore';
+import './css/index.css';
```
Comment (PrangonGhose) [inline src/main.jsx]: I believe there is no `./css/index.css` file.

**Review by PrangonGhose — CHANGES_REQUESTED** [review]
Hi @ShinhyoBelok  :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Gdiazdiaz — APPROVED** [review]
Hey @ShinhyoBelok nothing to add at the moment, time to merge!

[Optional] I wanna ask if we can avoid using the universal tags (like h1, h2,) to style elements when pair-programming as it can cause issues.

**Review by Gdiazdiaz — COMMENTED** [review]
Currently no changes need to be made, you can merge!

**Pull request merged**